package.json

```diff
@@ -88,7 +88,7 @@
     "prepare": "yarn run build",
     "download-tools": "ts-node scripts/download-tools.ts",
     "clean": "git clean -f -x ./node_modules ./dist ./coverage ./tools",
-    "build": "webpack --mode production && yarn lint",
+    "build": "if-env SKIP_LINT && webpack --mode production || webpack --mode production && yarn lint",
```

Comment:
Can we avoid making scripts more complicated here?
It looks like `if-env` comes with some particular command line. And requires developers to learn that if they want to change one of the most fundamental scripts locally.
Doesn't a simple if-condition work on the call to `yarn build` in CI?

Reply:
Alternative could also be to have something like `yarn build-ci` which doesn't chain the `yarn lint` and call both steps explicitly in CI.

Reply:
cleanups all the changes and added a new command `yarn build:no-lint`